### PR TITLE
Covid update 3 - New Portland conference dates

### PR DESCRIPTION
**Pull request by ericholscher**

### CODE REVIEW

File: docs/conf/portland/2020/news/covid-19-update-03.rst [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+.. post:: March 18, 2020
+   :tags: portland-2020, covid-19
+
+COVID-19 Update 03 - New Portland Conference Dates
+==================================================
+
+Hey Folks,
+
+We hope that you're doing okay. It's been a stressful couple weeks, and we wanted to send positive thoughts as we all attempt to get through this. `Please let us know`_ if there is anything we can do to help you.
```
Comment (thatdocslady) [inline docs/conf/portland/2020/news/covid-19-update-03.rst]:
```suggestion
We hope that you're doing okay. It's been a stressful few weeks, and we wanted to send positive thoughts as we all attempt to get through this. `Please let us know`_ if there is anything we can do to help you. 
```

File: docs/conf/portland/2020/news/covid-19-update-03.rst [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+We hope that you're doing okay. It's been a stressful couple weeks, and we wanted to send positive thoughts as we all attempt to get through this. `Please let us know`_ if there is anything we can do to help you. 
+
+The true power of community comes in the most trying of times. It's been amazing watching folks come together on :doc:`Slack </slack>`, and we're trying to plan more ways for folks to feel connected and supported.
+
+We also wanted to give you an update on the Portland conference. **We have a new date, August 9-11, 2020.**
+
+Given all the uncertainty about the future, we are being cautious in our approach with the conference.
+We're announcing the new date so that people can mark it on their calendars,
+but we're opening ticket sales again until we are certain that the event can proceed.
```
Comment (thatdocslady) [inline docs/conf/portland/2020/news/covid-19-update-03.rst]:
```suggestion
but we're waiting to open ticket sales again until we are certain that the event can proceed. 
```

File: docs/conf/portland/2020/news/covid-19-update-03.rst [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+The true power of community comes in the most trying of times. It's been amazing watching folks come together on :doc:`Slack </slack>`, and we're trying to plan more ways for folks to feel connected and supported.
+
+We also wanted to give you an update on the Portland conference. **We have a new date, August 9-11, 2020.**
+
+Given all the uncertainty about the future, we are being cautious in our approach with the conference.
+We're announcing the new date so that people can mark it on their calendars,
+but we're opening ticket sales again until we are certain that the event can proceed. 
+
+We know that having a firm dates means that not everyone will be able to come.
```
Comment (thatdocslady) [inline docs/conf/portland/2020/news/covid-19-update-03.rst]:
```suggestion
We know that changing the conference dates means that not everyone will be able to attend.
```

File: docs/conf/portland/2020/news/covid-19-update-03.rst [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+In order to more easily handle requests for changes, we've `created a form`_ to handle incoming ticket change requests.
```
Comment (thatdocslady) [inline docs/conf/portland/2020/news/covid-19-update-03.rst]:
```suggestion
In order to more easily handle requests for changes, we `created a form`_ to handle incoming ticket change requests. 
```

**Review by thatdocslady — COMMENTED** [review]
Looks good, a few nit-picky suggestions (apply at will)

**Review by mxsasha — APPROVED** [review]
LGTM, I pushed commits for a few places where we missed the date (most of those we already fixed in the Prague 2020 website to come from the config).

Another thought: I think we should add an explicit suggestion for people not to book travel yet.

**Comment by ericholscher** [comment]
> Another thought: I think we should add an explicit suggestion for people not to book travel yet.

Added this.

File: docs/conf/portland/2020/hike.rst [hunk 5]
```diff
@@ -30,7 +30,7 @@ from the venue to the park, or you can `take transit <https://www.google.com/map
 What to bring
 ~~~~~~~~~~~~~
 
-May in Portland can be interesting. It will probably be in the 60s, with a chance of rain. So, please bring:
+August in Portland can be interesting. It will probably be in the 60s, with a chance of rain. So, please bring:
```
Comment (plaindocs) [inline docs/conf/portland/2020/hike.rst:33]: THIS IS HILARIOUS

File: docs/conf/portland/2020/news/covid-19-update-03.rst [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+.. post:: March 18, 2020
+   :tags: portland-2020, covid-19
+
+COVID-19 Update 03 - New Portland Conference Dates
+==================================================
+
+Hey Folks,
+
+We hope that you're doing okay. It's been a stressful few weeks, and we wanted to send positive thoughts as we all attempt to get through this. `Please let us know`_ if there is anything we can do to help you.
```
Comment (plaindocs) [inline docs/conf/portland/2020/news/covid-19-update-03.rst:11]: <3

**Review by plaindocs — APPROVED** [review]
Looks great.

File: docs/conf/portland/2020/hike.rst (hunk 5 above)
Comment (mxsasha) [reply docs/conf/portland/2020/hike.rst:33]: Oh yeah, I did kind of a blanket update.

**Comment by mopdx** [comment]
Looks good.

**Pull request merged**